WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewController.swift

```diff
@@ -46,6 +46,9 @@ final class ShippingLabelFormViewController: UIViewController {
     }
 
     override var shouldShowOfflineBanner: Bool {
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) {
+            return false
```

Comment:
just wondering, when do we want to not show offline banner?

Reply:
Sorry for not explaining clearer 😅 The reason is that with the split view in place, I'm [showing the offline banner on the split view wrapper controller](https://github.com/woocommerce/woocommerce-ios/blob/abddae029a69d8fcc37b4ab1f68fa0ba6acd3d62/WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift#L29) instead of in child view controllers to avoid unnecessary setup and weird animations.